.github/actions/gke/test-config-helm.yaml

```diff
@@ -12,3 +12,7 @@ config:
   - type: "tunnel-ipsec"
     index: 4
     cilium-install-opts: "--helm-set=encryption.enabled=true --helm-set=encryption.type=ipsec --datapath-mode=tunnel"
+  - type: "tunnel-ingress-controller"
+    index: 5
+    cilium-install-opts: "--helm-set=kubeProxyReplacement=true --helm-set=ingressController.enabled=true"
+    nodes: 1
```

Comment:
Does this test provide any additional coverage compared to the ones already present? If so, could it be replaced by a standard kind-based test? I think that the general idea is to limit as much as possible cloud-based tests, as typically slower/more flaky/more expensive.

Reply:
>  If so, could it be replaced by a standard kind-based test? I think that the general idea is to limit as much as possible cloud-based tests, as typically slower/more flaky/more expensive.

Agreed, it's exactly what I have done so far, all the ingress/gateway/l7 LB tests are now with kind clusters, so this mini cluster (with 1 node only) is intentional, as we don't have any Ingress tests running in any cloud environment now.

Reply:
Sounds good to me. Could you maybe just extend the commit message a bit to clarify the reasons? So that we don't risk removing it in the future.

Reply:
thanks and done, commit message is updated.